[2024-08-16T22:08:49.332+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-08-16T22:08:49.343+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: predict_values_TS_6.create_s3_bucket scheduled__2024-08-04T00:00:00+00:00 [queued]>
[2024-08-16T22:08:49.348+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: predict_values_TS_6.create_s3_bucket scheduled__2024-08-04T00:00:00+00:00 [queued]>
[2024-08-16T22:08:49.349+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-08-16T22:08:49.358+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): create_s3_bucket> on 2024-08-04 00:00:00+00:00
[2024-08-16T22:08:49.362+0000] {warnings.py:110} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=2865) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-08-16T22:08:49.363+0000] {standard_task_runner.py:63} INFO - Started process 2869 to run task
[2024-08-16T22:08:49.363+0000] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'predict_values_TS_6', 'create_s3_bucket', 'scheduled__2024-08-04T00:00:00+00:00', '--job-id', '14', '--raw', '--subdir', 'DAGS_FOLDER/etl.py', '--cfg-path', '/tmp/tmpqgisq_3i']
[2024-08-16T22:08:49.365+0000] {standard_task_runner.py:91} INFO - Job 14: Subtask create_s3_bucket
[2024-08-16T22:08:49.396+0000] {task_command.py:426} INFO - Running <TaskInstance: predict_values_TS_6.create_s3_bucket scheduled__2024-08-04T00:00:00+00:00 [running]> on host 08898c2f83c5
[2024-08-16T22:08:49.450+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Ruslan Solarev' AIRFLOW_CTX_DAG_ID='predict_values_TS_6' AIRFLOW_CTX_TASK_ID='create_s3_bucket' AIRFLOW_CTX_EXECUTION_DATE='2024-08-04T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-08-04T00:00:00+00:00'
[2024-08-16T22:08:49.451+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-08-16T22:08:49.465+0000] {base.py:84} INFO - Using connection ID 'minio_conn' for task execution.
[2024-08-16T22:08:49.465+0000] {connection_wrapper.py:387} INFO - AWS Connection (conn_id='minio_conn', conn_type='aws') credentials retrieved from extra.
[2024-08-16T22:08:49.465+0000] {warnings.py:110} WARNING - <string>:9: AirflowProviderDeprecationWarning: extra['host'] is deprecated and will be removed in a future release. Please set extra['endpoint_url'] instead

[2024-08-16T22:08:49.664+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-08-16T22:08:49.665+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-08-16T22:08:49.672+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=predict_values_TS_6, task_id=create_s3_bucket, run_id=scheduled__2024-08-04T00:00:00+00:00, execution_date=20240804T000000, start_date=20240816T220849, end_date=20240816T220849
[2024-08-16T22:08:49.697+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-08-16T22:08:49.707+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-08-16T22:08:49.710+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
